### PR TITLE
Php 8.0

### PR DESCRIPTION
**Pull request by syteknet-core**
* php 8 compatibility for composer
* php8-0 update for ci

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 || ^8.0",
```
Comment (WyriHaximus) [inline composer.json:17]: Might as well bump to 7.4 minimum

```suggestion
        "php": "^7.4 || ^8.0",
```

**Review by WyriHaximus — CHANGES_REQUESTED** [review]
Could you also include the `composer.lock` changes? `composer update -W` should do the trick

File: composer.json (hunk 1 above)
Comment (syteknet-core) [reply composer.json]: good hint! requirement bumped to 7.4 minimum and the composer.lock is also updated

Comment (WyriHaximus) [reply composer.json]: :+1:

**Comment by WyriHaximus** [comment]
Awesome thanks :+1:

**Pull request merged**